src/NLog/Layouts/Log4JXmlEventLayout.cs

```diff
@@ -58,6 +58,26 @@ public Log4JXmlEventLayout()
         public Log4JXmlEventLayoutRenderer Renderer { get; private set; }
 
         /// <summary>
+        /// Gets or sets a value indicating whether to include contents of the <see cref="MappedDiagnosticsContext"/> dictionary.
+        /// </summary>
+        /// <docgen category='Payload Options' order='10' />
+        public bool IncludeMdc { get { return Renderer.IncludeMdc; } set { Renderer.IncludeMdc = value; } }
```

Comment:
No idea. Just thought it was weird that the Layout didn't have access to all the fancy options on the layout-renderer.